Shared/IISFunctions/Get-IISModules.ps1

```diff
@@ -147,11 +159,12 @@ function Get-IISModules {
                                         if ($null -ne $signature.SignerCertificate.Subject) {
                                             Write-Verbose "Signer information found. Subject: $($signature.SignerCertificate.Subject)"
                                             $signatureDetails.Signer = $signature.SignerCertificate.Subject.ToString()
-                                            $signatureDetails.IsMicrosoftSigned = $signature.SignerCertificate.Subject -cmatch "O=Microsoft Corporation, L=Redmond, S=Washington"
+                                            $signatureDetails.IsMicrosoftSigned = $signature.SignerCertificate.Subject -cmatch $signerSubject
                                         }
                                     }
                                 } else {
                                     Write-Verbose "No signature information found for module $($m.name)"
+                                    $isModuleSigned = $false
```

Comment:
Not required to be here, but fine to have.

Reply:
Indeed. This is a leftover which I forgot to remove. Will remove it together with the environment variable adjustment